src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedCasesService.kt

```diff
@@ -43,15 +43,21 @@ class UnallocatedCasesService(
     return hmppsTierApiClient.getTierByCrn(crn)
   }
 
-  fun getProbationStatus(crn: String): String {
+  fun getProbationStatus(crn: String): ProbationStatus {
     val activeConvictions = communityApiClient.getActiveConvictions(crn).size
     return when {
-      activeConvictions > 1 -> "Currently managed"
```

Comment:
Not sure about this when construct. We have some logic based on size on 2 different place that I guess should be encapusalte din functions with a menaingfulName that describe why >1 is relevant .